src/pages/gen2/reference/amplifyconfiguration/index.mdx

```diff
@@ -16,3 +16,43 @@ In Amplify (Gen 2), the CLI will generate an `amplifyconfiguration.json` file wi
 You can also manually create this file for a specified Amplify app ID and branch, or an AWS CloudFormation stack name with [`amplify generate config`](/gen2/reference/cli-commands#amplify-generate-config).
 
 {/* @TODO full type/interface to reference */}
+
+## Extending Amplify configuration file
+
+The `amplifyconfiguration.json` file is not just a static artifact; it's designed to be extendable to suit the evolving needs of your application. By leveraging the `addOutput` method you can programmatically add or modify configurations.
+This is particularly useful for customizing outputs that are not directly exposed through the Amplify constructs or for dynamically adjusting your app's configuration in response to changes in your backend strategy.
```

Comment:
```suggestion
The `amplifyconfiguration.json` file is not just a static artifact; it's designed to be extendable to suit the evolving needs of your application. By leveraging the `addOutput` method you can programmatically add or modify configurations. This is particularly useful for customizing outputs that are not directly exposed through the Amplify constructs or for dynamically adjusting your app's configuration in response to changes in your backend strategy.
```

was this an intentional line break?

Reply:
yes, for readability but can change it to combined.

Reply:
ah if it renders as two separate paragraphs can we separate in the markdown with a newline for readability?

Reply:
keeping it as one paragraph should be fine.